content/en/docs/restaurant/reference/restaurant_card/index.md

```diff
@@ -22,10 +22,10 @@ You can define the particularities that is associated with a Restaurant, such as
 
 #### Restaurant configuration examples
 
-- **Restaurant** - This is an à la carte restaurant in which payment is carried out after dining. It has more than one kitchen that prepares the food. 
-- **Burger Shop** - Payment is carried out as soon as the order is placed, before serving. It has one kitchen in the back area. 
-- **Pizzeria** - Payment is carried out as soon as the order is placed, before serving. The food is prepared in front of the customer.
-- **Take-Away Kiosk** - Payment is carried out as soon as the order is placed, with food being prepared in front of the customer. There's no seating arrangement.
+- **Fine-dining restaurant** - This is an à la carte restaurant in which payment is carried out after dining. It has more than one kitchen that prepares the food. 
```

Comment:
`It has more than one kitchen that prepares the food.`
did you mean "kitchen station"?